tv_apps/app-deck/build/build.js

```diff
@@ -4,10 +4,25 @@
 
 var utils = require('utils');
 
+var AppDeckBuilder = function() {};
+
+// XXX: we should extract the logic of copying default-cards.json
+// to one common place. See https://bugzil.la/1123585
+AppDeckBuilder.prototype.execute = function(options) {
+  var stageDir = utils.getFile(options.STAGE_APP_DIR);
+  var configFile = utils.getFile(stageDir.path, 'js', 'init.json');
+  var defaultConfigFile = utils.getFile(options.GAIA_DIR, 'tv_apps',
+                                'tv_shared', 'resources', 'default-cards.json');
+  var defaultConfig = utils.getJSON(defaultConfigFile);
+  // TODO: to verify correctness of defaultConfig
```

Comment:
I don't think we need this TODO. There few types of verification can be done here:
1. content correctness: This depends on the code of app-deck. Once we changed the code, this part needs to change.
2. syntax correctness: The utils.getJSON already checks this for you.

If the verification here is the first case, I don't think we need to do it because programmer can know it once opening it.